harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -698,60 +769,62 @@ def __init__(self, trial_context: PyTorchTrialContext) -> None:
         """
         Initializes a trial using the provided trial context.
 
-        Override this function to initialize any shared state between the
-        function implementations.
+        1. Initialize models, optimizers, and LR schedulers and wrap them with ``context.Model``,
```

Comment:
A couple of points.  First, let's use the restructured text syntax for numbered list:
```
#. some text
#. more text
```

Second, this numbered list is missing some context as to what it represents conceptually.  Maybe change the first line to something like:

```
"""
Initialize a trial using the provided trial context.  The general steps are:

#. Initialize model(s) and wrap them with ``context.Model``.
#. Initialize optimizer(s) and wrap them with ``context.Optimizer``.
#. Initialize learning rate schedulers and wrap them with ``context.LRScheduler``.
#. If desired, wrap models and optimizer with ``context.configure_apex_amp`` 
   to use ``apex.amp`` for automatic mixed precision.

Here is a code example:
...
```